### PR TITLE
Fix js & css coding style

### PR DESCRIPTION
**Pull request by chihsuan**

### CODE REVIEW

File: website/js/average-words.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 function addVisWord(media, avgWords) {
-  var selector = ".avgWords-container ." + media+'-container';
-  var header = '<h3>' + mediaEN2C[media] + '<\/h3><span>' + avgWords +
+  var media_Chinese = mediaEN2C[media];
```
Comment (chihsuan) [inline website/js/average-words.js]: `media_Chinese` should be `mediaChinese` or `mediaChineseName`

File: website/js/cloud.js [hunk 2]
```diff
@@ -4,154 +4,153 @@
...
+    var cloud = svg.selectAll("g text")
+      .data(words, function(d) {
+        return d.text;
+      })
+
+    var getColor = function(d) {
+        max = 0
+        dict = {}
+        for (var i in media) {
+          tmp = report['words_count'][d.index][3][media[i]]['sum'] / report[media[i]]['news_count']
```
Comment (chihsuan) [inline website/js/cloud.js]: I think this line is too long and difficult to understand. It' would be great to break into segments.

File: website/js/cloud.js [hunk 3]
```diff
@@ -4,154 +4,153 @@
...
-                    key = media[i]
-                }
-            }
-            return mediaColor[key]
+  var fill = d3.scale.category20();
+  var nav_width = 240
+  var width
+  var marginCloud = 80
+
+  if ($(window).width() > 980) {
```
Comment (chihsuan) [inline website/js/cloud.js]: We should replace 980 by using a constant, such as `smallDesktopWidthSize`

File: website/js/cloud.js [hunk 4]
```diff
@@ -4,154 +4,153 @@
...
+        p1ClearCards()
+        news = report['words_count'][d.index][2]
+        dict = {}
+        for (i in mediaEN) {
+          dict[mediaEN[i]] = 0
+        }
+        for (var i in news) {
+          media = mediaC2EN[news[i]['media']]
+          dict[media]++
+            console.log(news[i]['title'] + " " + media)
```
Comment (chihsuan) [inline website/js/cloud.js]: We should not commit the console.log.

File: website/js/cloud.js [hunk 5]
```diff
@@ -4,154 +4,153 @@
...
+      .attr('cursor', 'pointer')
+      .text(function(d) {
+        return d.text;
+      }).on("click", function(d) {
+        // createTimeline('#timeline', data)
+        // add word collection
+        var offset = $('#timeline').offset()
+        $('body,html').animate({ scrollTop: offset.top - 25 }, 'slow');
+
+        p1ClearCards()
```
Comment (chihsuan) [inline website/js/cloud.js]: What's the meaning of p1?

File: website/js/average-words.js (hunk 1 above)
Comment (chihsuan) [reply website/js/average-words.js]: Also, should not use the global variable. The `mediaEN2C` should be passed by the arguments.

File: website/js/cloud.js [hunk 6]
```diff
@@ -4,154 +4,153 @@
...
 
 
 function showNewWords(vis, i) {
-    max = report['words_count'][0][1]
-    scale = max / 100
-    cloudConfig =  report['words_count'].map(function(obj,index) {
-        return { text: obj[0], size: (10 + obj[1] / scale),index:index};
-    });
-    vis.update(cloudConfig)
+  max = report['words_count'][0][1]
```
Comment (chihsuan) [inline website/js/cloud.js]: If the key is constant, it's better to use dot to get the value of object, e.g report.worlds_count[0][1].

**Comment by ching56** [comment]
the preceding problems might be solved

File: website/js/title-analysis.js [hunk 7]
```diff
@@ -1,19 +1,19 @@
-function p2AddNewsCard(media, header, detail) {
-    var selector = "#2th-collection .card-container ." + media;
-    var content = "<div class=\"card\">\
+function titleAnalysisAddNewsCard(media, header, detail) {
+  var selector = "#2th-collection .card-container ." + media;
+  var content = "<div class=\"card\">\
```
Comment (chihsuan) [inline website/js/title-analysis.js]: It is recommended to use single quotes `''` for strings in JS. And the whitespace at the beginning of each line can't be safely stripped at compile time; whitespace after the slash will result in tricky errors. Use + for string concatenation instead:

File: website/js/title-analysis.js [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
-function p2ClearCards(media, header, detail) {
-    $("#2th-collection .card-container .list").each(function(index) {
-      var header = $(this).find('h3').text()
-        $(this).html('<h3>' + header + '<\/h3>')
-    });
-    refreshCards()
+function titleAnalysisClearCards(media, header, detail) {
+  $("#2th-collection .card-container .list").each(function(index) {
+    var header = $(this).find('h3').text()
+    $(this).html('<h3>' + header + '<\/h3>')
```
Comment (chihsuan) [inline website/js/title-analysis.js]: Actually, we don't need to escape the `/` character.
-> '<h3>' + header + '</h3>'

**Pull request merged**